src/ui/components/IdentifierOptions/IdentifierOptions.tsx

```diff
@@ -119,7 +120,7 @@ const IdentifierOptions = ({
 
   const share = async () => {
     await Share.share({
-      text: cardData.displayName + " " + cardData.id,
+      text: oobi,
```

Comment:
Hmm unless I'm misunderstanding, this goes straight into the native share option right? Whereas it should actually be the third place that can open `<ShareConnection />`